src/panels/lovelace/common/structs/is-entity-id.ts

```diff
@@ -7,11 +7,22 @@ const isEntityId = (value: unknown, context: StructContext): StructResult => {
   if (!value.includes(".")) {
     return [
       context.fail({
-        type: "entity id should be in the format 'domain.entity'",
+        type: "entity ID should be in the format 'domain.entity'",
       }),
     ];
   }
   return true;
 };
 
 export const EntityId = struct("entity-id", isEntityId);
+
+const isEntityIdOrAll = (
+  value: unknown,
+  context: StructContext
+): StructResult => {
+  if (typeof value === "string" && value === "all") return true;
+
+  return isEntityId(value, context);
+};
+
+export const EntityIdOrAll = struct("entity-id", isEntityIdOrAll);
```

Comment:
Haven't checked the docs, but I guess it should be:
```suggestion
export const EntityIdOrAll = struct("entity-id-all", isEntityIdOrAll);
```